common/cpp/src/google_smart_card_common/tuple_unpacking.h

```diff
@@ -15,6 +15,8 @@
 #ifndef GOOGLE_SMART_CARD_COMMON_TUPLE_UNPACKING_H_
 #define GOOGLE_SMART_CARD_COMMON_TUPLE_UNPACKING_H_
 
+#include <stddef.h>
```

Comment:
Is there a reason to prefer this over `#include <cstddef>`?

Reply:
Answering myself: The style guide does not recommend one over the other, as long as it is consistent. Looking at a few adjacent files, the majority uses this style.
However, I found `#include <cstdarg>` in chromeos_smart_card_connector/common/cpp/src/google_smart_card_common/formatting.h, so perhaps that could be changed.

Reply:
I believe, technically, cstddef provides only `std::size_t` and not `size_t`, meanwhile the code in this file uses the latter. I personally find it a bit annoying to write `std::` in front of such "almost" built-in type, but this is not a strong argument :)
WDYT?

Reply:
I don't have a strong opinion for either here.

Reply:
Let's go with ".h" then, for now.